app/templates/gulpfile.js

```diff
@@ -451,6 +451,12 @@ gulp.task('serve-dev', ['build'], function() {
   return serve('dev' /*env*/);
 });
 
+gulp.task('develop', ['inject','styles','less-watcher'], function() {
+    //skip the build and the browser sync
+    var nodeOptions = getNodeOptions('local');
+    $.nodemon(nodeOptions);
+});
+
```

Comment:
Sounds like a short-hand for `gulp serve-local --nosync`. Would a developer not prefer using browser sync most of the time?